webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/servers/RemoveNodeService.scala

```diff
@@ -202,10 +203,13 @@ class RemoveNodeServiceImpl(
           for {
             moved  <- groupLibMutex.writeLock {atomicDelete(nodeId, modId, actor) } ?~! "Error when deleting a node"
             closed <- nodeConfigurationsRepo.closeNodeConfigurations(nodeId)
-
             invLogDetails = InventoryLogDetails (nodeInfo.id,nodeInfo.inventoryDate, nodeInfo.hostname, nodeInfo.osDetails.fullName, actor.name)
             eventlog = DeleteNodeEventLog.fromInventoryLogDetails (None, actor, invLogDetails)
             saved  <- actionLogger.saveEventLog(modId, eventlog)
+                      // if the node was a policy server, we need to delete group and directive/rule for it
+            _      <- if(nodeInfo.isPolicyServer) {
+                        policyServerManagement.deleteRelaySystemObjects(nodeId) ?~! s"Error when deleting system objects (groups, directives, rules) related to relay server '${nodeId.value}'"
```

Comment:
this method is missing a logging
it is called by actualNodeDeletion, which returns a Box[JValue], and error is only poped up to the UI or API response